tracker/tracker_debugger_api.yaml

```diff
@@ -225,9 +225,10 @@ components:
             - 16
             - 17
             - 18
+            - 19
+            - 20
+            - 21
```

Comment:
I'm not sure if it's better to use enum here or int with min/max values
![event_type](https://user-images.githubusercontent.com/4488774/122076960-9bd75d00-cdfb-11eb-9e81-f9795255b8f2.png)

Reply:
We'll have holes in the range, e.g. events 10 and 11. Thanks for reminding me though, I've commented these two out and added comments naming all event types.